alignak/daemon.py

```diff
@@ -198,6 +199,8 @@ def __init__(self, name, config_file, is_daemon, do_replace, debug, debug_file):
         self.debug = debug
         self.debug_file = debug_file
         self.interrupted = False
+        self.pidfile = None
+        self.http_interface = GenericInterface(object)
```

Comment:
Well if pylnt complains about tye then we should respect this type. I was more concern about memory consumption and unuesed object in memory